docs/get_started/get_started_demos.md

```diff
@@ -1,48 +1,45 @@
 # Basic OpenVINO™ Workflow {#openvino_docs_get_started_get_started_demos}
 
-This guide will walk you through a basic workflow for Intel® Distribution of OpenVINO™ toolkit, including how to use code samples.
+The guide walks through a basic workflow for building and running C++ code samples in OpenVINO. This guide works for C++ samples only and is not intended for use with the Python samples.
 
-This guide assumes you have completed all the installation and preparation steps. If you have not, check out the <a href="prerequisites">Prerequisites</a> section to install OpenVINO Runtime, install OpenVINO Development Tools, or build samples and demos.
+To work through this guide, you must first install OpenVINO Runtime, install OpenVINO Development tools, and build the sample applications. See the <a href="#prerequisites">Prerequisites</a> section for instructions on how to do so.
 
-After that, you will perform the following steps:
+Once the prerequisites have been installed, perform the following steps:
 
 1. <a href="#download-models">Use Model Downloader to download a suitable model.</a>
 2. <a href="#convert-models-to-intermediate-representation">Convert the model with Model Optimizer.</a> 
 3. <a href="#download-media">Download media files to run inference.</a>
-4. <a href="#run-image-classification">Run inference on a sample and see the results.</a> The following code sample is used as an example:
-    - <a href="#run-image-classification">Image Classification Code Sample</a>
+4. <a href="#run-image-classification">Run inference with the Image Classification sample application and see the results.</a>
 
 ## <a name="prerequisites"></a>Prerequisites
 
 ### Install OpenVINO Runtime
 
-If you have not yet installed and configured the toolkit, see the following guides:
+If you have not yet installed and configured OpenVINO Runtime, see the following guides. The source code for the OpenVINO samples is not included with the PyPI distribution, so the archive file installation method must be used.
```

Comment:
```suggestion
If you have not yet installed and configured OpenVINO Runtime, see the following guides. The source code for OpenVINO samples is not included with the PyPI distribution, so the archive file installation method must be used.
```

Reply:
If by "Github installations" you mean building [OpenVINO from source](https://github.com/openvinotoolkit/openvino/wiki/BuildingForLinux) by cloning the openvino repository, then yes, the samples are included. I'm not sure about the Docker installation.

Reply:
yeah...I checked - building from source includes the step for building samples, and Docker installation seems also including samples as the installation pages has described how to run samples. So this sentence here may need be adjusted a little bit.

Reply:
Thanks @kblaszczak-intel , I like your suggestion! I went ahead and changed it to match what you said.